src/little-bang-parser/little_bang_generated_parser.mly

```diff
@@ -30,6 +31,7 @@ open Tiny_bang_parser_support;;
 %nonassoc EQUALITY LESS_THAN
 (* %right '<-' *)
 (* %left '+' '-' *)
+%nonassoc KEYWORD_REF
```

Comment:
That depends on the language (Little Bang/Tiny Bang have a different meaning of the `ref` keyword than Big Bang does)...

In Big Bang, `ref` is a variable declaration, so only (1) makes any sense. In Little Bang, `ref` is a constructor, so (2) makes more sense.